packages/infra/lib/lambdas-nested-stack.ts

```diff
@@ -578,4 +592,35 @@ export class LambdasNestedStack extends NestedStack {
 
     return acmCertificateMonitorLambda;
   }
+
+  private setupRosterUploadLambda(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    envType: EnvType;
+    hl7v2RosterBucket: s3.Bucket;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
+    const { lambdaLayers, vpc, sentryDsn, envType, alarmAction, hl7v2RosterBucket } = ownProps;
+
+    const hl7v2RosterUploadLambda = createLambda({
+      stack: this,
+      name: "Hl7v2RosterUpload",
+      runtime: lambda.Runtime.NODEJS_18_X,
```

Comment:
If no specific need for Node18, might as well leave as the default runtime so it's easier to upgrade later?